cicd/forgeops-tests/lib/utils/kubectl.py

```diff
@@ -0,0 +1,61 @@
+# Copyright (c) 2016-2019 ForgeRock AS. Use of this source code is subject to the
+# Common Development and Distribution License (CDDL) that can be found in the LICENSE file
+
+# Lib imports
+import subprocess
+
+# Framework imports
+from utils import logger
+
+KUBECTL_COMMAND = "kubectl"
+
```

Comment:
You guys prolly know this - but there is a native python library for talking to k8s
No need for kubectl

Reply:
yep - that is the one. It might be more robust / easier vs. shelling to kubectl